man/rmd/overview.Rmd

```diff
@@ -1,32 +1,24 @@
 
-Tidyverse selections implement a dialect of R where operators make
-it easy to select variables:
+tidyselect implements a DSL for selecting variables. It provides helpers
+for selecting variables:
 
-- `:` for selecting a range of consecutive variables.
-- `!` for taking the complement of a set of variables.
-- `&` and `|` for selecting the intersection or the union of two
-  sets of variables.
-- `c()` for combining selections.
+- `var1:var10`: variables between `var1` on the left and `var2` on the right.
```

Comment:
```suggestion
- `var1:var10`: variables between `var1` on the left and `var10` on the right.
```

Reply:
I'm not sure what you mean by "on the left" and "on the right". I guess this is just referring to the LHS and RHS of `:`, but I think someone reading this without prior knowledge would wonder if it was about something else. How about:

`var1:var10`: consecutive variables between (and including) `var1` and `var10`.